### PR TITLE
all: update atlas

### PR DESCRIPTION
**Pull request by zeevmoney**

### CODE REVIEW

File: internal/entimport/import_test.go [hunk 1]
```diff
@@ -3930,6 +3917,14 @@ type Inspector struct {
 	mock.Mock
 }
 
+func (_m *Inspector) InspectTable(ctx context.Context, name string, opts *schema.InspectTableOptions) (*schema.Table, error) {
+	return nil, nil
```
Comment (masseelch) [inline internal/entimport/import_test.go:3921]: What is the point of an empty body?

File: internal/entimport/import_test.go [hunk 2]
```diff
@@ -3930,6 +3917,14 @@ type Inspector struct {
...
+}
+
+func (_m *Inspector) InspectRealm(ctx context.Context, opts *schema.InspectRealmOption) (*schema.Realm, error) {
+	return nil, nil
```
Comment (masseelch) [inline internal/entimport/import_test.go:3925]: Same here.

File: internal/entimport/import_test.go (hunk 1 above)
Comment (zeevmoney) [reply internal/entimport/import_test.go:3921]: it's a mock, I must implmenet the interface and there is no usage for this func.

File: internal/entimport/import_test.go (hunk 2 above)
Comment (zeevmoney) [reply internal/entimport/import_test.go:3925]: it's a mock, I must implmenet the interface and there is no usage for this func.

File: internal/entimport/import_test.go (hunk 1 above)
Comment (masseelch) [reply internal/entimport/import_test.go:3921]: :man_facepalming: sorry

**Review by rotemtam — APPROVED** [review]

**Pull request merged**